numba/targets/mathimpl.py

```diff
@@ -20,9 +20,11 @@
 
 
 # Helpers, shared with cmathimpl.
+FLT_MAX = 3.4028234663852886E+38
+FLT_MIN = 1.1754943508222875E-38
 
-FLT_MAX = 3.402823466E+38
-FLT_MIN = 1.175494351E-38
+DBL_MAX = 1.7976931348623157E+308
+DBL_MIN = 2.2250738585072014E-308
```

Comment:
i thk these should use `sys.float_info.max` and `sys.float_info.min` instead of hardcoding it.

Reply:
Or, use `np.finfo` so we can fix the single-precision ones two.

Reply:
I thought about that but wanted to be consistent across float and double declarations. Am happy either way.

Reply:
Add `np.finfo` call in df4bdea.